src/ontology/mondo-ingest.Makefile

```diff
@@ -560,7 +554,20 @@ $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv: $(TMPDIR)/nord.tsv config/external-conte
 .PHONY: external-content-nord
 external-content-nord: $(EXTERNAL_CONTENT_DIR)/nord.robot.owl
 
-update-externally-managed-content: external-content-nord
+tmp/ordo-subsets.tsv: $(COMPONENTSDIR)/ordo.owl
+	$(ROBOT) query -i $< --query ../sparql/select-ordo-subsets.sparql $@
+
+$(EXTERNAL_CONTENT_DIR)/ordo-subsets.robot.template.tsv: tmp/ordo-subsets.tsv tmp/mondo.sssom.tsv
+	python3 $(SCRIPTSDIR)/ordo_subsets.py \
+	--mondo-mappings-path tmp/mondo.sssom.tsv \
+	--class-subsets-tsv-path tmp/ordo-subsets.tsv \
+	--onto-config-path metadata/ordo.yml \
+	--outpath $@
+
+.PHONY: external-content-ordo
+external-content-ordo: $(EXTERNAL_CONTENT_DIR)/ordo-subsets.robot.template.tsv
+
+update-externally-managed-content: external-content-nord external-content-ordo
```

Comment:
## Added to build
In order to do #543, I needed to address 2/3 of the sub-tasks in #515:
- Add this output to the `external-content` part of the pipeline
- Which, in turn, adds this output to `build-mondo-ingest`

Reply:
Note: Nico has reviewed and approves.